### ISSUE TITLE
VS Code 1.14 release will block SVGs from http:// sources

### ISSUE THREAD

**Issue by gregvanl** [Jason-Rev/vscode-spell-checker#103]
For security concerns, VS Code 1.14 will block extension README and CHANGELOG SVG images coming from http:// sources. 

See https://github.com/Microsoft/vscode-docs/blob/vnext/release-notes/v1_14.md#changes-for-extension-authors for details.

![spell svg blocked](https://user-images.githubusercontent.com/12818376/27715741-10a26ac2-5cee-11e7-9d3b-f385f5e731c9.png)

The README badges just need to be updated to https://

```markdown
# Spelling Checker for Visual Studio Code
[![Current Version](http://vsmarketplacebadge.apphb.com/version/streetsidesoftware.code-spell-checker.svg)](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)
[![Install Count](http://vsmarketplacebadge.apphb.com/installs/streetsidesoftware.code-spell-checker.svg)](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)
[![Open Issues](http://vsmarketplacebadge.apphb.com/rating/streetsidesoftware.code-spell-checker.svg) ](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)
```

**Comment by Jason3S** [comment]
Thank you. I'll fix that.

-Jason

**Comment by Jason3S** [comment]
I just published a fix.

**Comment by gregvanl** [comment]
Thanks for jumping on this. Really like your extension and I wanted to be sure the README looked good on the next release.

**Comment by Jason3S** [comment]
Thank you for pointing it out. I would have never noticed.
